python/mxnet/gluon/data/vision/transforms.py

```diff
@@ -581,3 +582,28 @@ def hybrid_forward(self, F, x):
         if is_np_array():
             F = F.npx
         return F.image.random_lighting(x, self._alpha)
+
+
+class RandomApply(Sequential):
+    """Apply a list of transformations randomly given probability
+
+    Parameters
+    ----------
+    Inputs:
+        - **transforms**: list of transformations
+        - **p**: probability
```

Comment:
- aren't these two supposed to be parameters? also it would be useful to specify the datatype of parameters( eg. float for probability). 
 - inputs will be the input image( data).

( as a reference please see other transforms in line 562)

Reply:
Yes. Updated.